nilearn/signal.py

```diff
@@ -407,8 +410,8 @@ def _ensure_float(data):
 
 
 def clean(signals, sessions=None, detrend=True, standardize='zscore',
```

Comment:
maybe we should deprecate `sessions` here, so that we can use  `runs` instead in a future release (0.9).

Reply:
Sounds good. May I ask if there's an example for implementing deprecation warning in nilearn? I have never handled this kind of things before.